packages/server/package.json

```diff
@@ -11,7 +11,7 @@
     "start": "nest start",
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
+    "start:prod": "node dist/src/main",
```

Comment:
This shouldn't have been necessary and is indicative of a bad build configuration. It's fine for now, I'll look into this later when we get closer to production.